packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import contentStyles from "./ContentStyles.styles";
+import {
+  confirmStyles,
+  confirmHeaderStyles,
+  confirmDescriptionStyles,
+  confirmFooterStyles,
+} from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.node.isRequired,
+  description: PropTypes.node,
+  isOpen: PropTypes.bool.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  onClose: PropTypes.func,
+  onConfirm: PropTypes.func.isRequired,
+  heading: PropTypes.node.isRequired,
+};
+
+const defaultProps = {
+  buttonSize: ShirtSizes.MEDIUM,
+  confirmButtonType: "destructive",
+  description: null,
+  onClose: () => {},
+};
+
+const Confirmation = props => {
+  const {
+    heading,
+    buttonSize,
+    confirmButtonType,
+    confirmLabel,
+    description,
+    onConfirm,
+    onCancel,
+    onClose,
+    isOpen,
+  } = props;
+  const confirmButtonRef = React.useRef(null);
+
+  React.useEffect(() => {
+    if (confirmButtonRef.current) confirmButtonRef.current.focus();
+  }, [confirmButtonRef]);
+
+  const I18n = useI18n();
+  return (
+    <Popover isOpen={isOpen} onClose={onClose}>
```

Comment:
In order for this component to be used _without_ a `<DropdownMenu>` component (like https://github.com/acl-services/paprika/issues/156), I think it's going to need a `getPositioningElement` prop for the `<Popover>`... isn't it?

Reply:
Good point I'll look into this

Reply:
I think this might be a good pattern to follow in this case:
https://github.com/acl-services/paprika/blob/master/packages/DatePicker/src/DatePicker.js#L91